src/librustc_back/rpath.rs

```diff
@@ -40,10 +40,7 @@ pub fn get_rpath_flags<F, G>(config: RPathConfig<F, G>) -> Vec<String> where
     debug!("preparing the RPATH!");
 
     let libs = config.used_crates.clone();
-    let libs = libs.into_iter().filter_map(|(_, l)| {
-        l.map(|p| p.clone())
-    }).collect::<Vec<_>>();
-
+    let libs = libs.into_iter().filter_map(|(_, l)| l).collect::<Vec<_>>();
```

Comment:
fwiw `let libs: Vec<_> = ...` is more en-vogue right now. 

Also what happened here? The clone is just totally gone? I'm guessing the clone was totally redundant since this is a by-value iter?